Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "eem"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Is there any chance to build this project with old edition?
Currently I built this project on host not inside container. It requires to update. But `rustup` doesn't work properly in container.

Reply:
@0xDen You need to pull latest changes from https://github.com/fobnail/fobnail-sdk and rebuild container. I have already updated Rust to the latest version.

There is no problem with building this project outside of the container. Actually, it needs to buildable both inside and outside of container since we want to publish it to crates.io

Reply:
@arturkow2000 Yep. Thanks a lot for updating. (I've started to update it myself)